src/main/java/pacman/ghost.java

```diff
@@ -4,4 +4,13 @@ public ghost (int x, int y,boolean exist){
 		super(x,y,exist);
         this.eatable = false;
 	}
+	protected boolean getEatable(){
+
+		return this.eatable;
```

Comment:
The name '"eatable" isn't much descriptive and little specific.

Reply:
Thanks, the function has been changed to isEatable()